orchestrator/modules/tools/discovery/handlers_autonomy.py

```
@@ -48,19 +48,44 @@ async def get_autonomy_level(
+    )
 
     level = params.get("level")
     if not level:
         return {"success": False, "error": "level is required (standard | full)"}
 
+    # Actor + prior level, captured before the write for the audit trail. The
+    # executor re-mints _agent_id server-side, so it can't be spoofed by the LLM.
+    actor = params.get("_agent_id")
+    try:
+        previous = _get(db, workspace_id)
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)